src/AnalysisPrograms/Recognizers/PteropusSpecies.cs

```diff
@@ -0,0 +1,450 @@
+
+            // now construct the standard decibel spectrogram WITH noise removal, and look for LimConvex
+            // get frame parameters for the analysis
+            var sonogram = (BaseSonogram)new SpectrogramStandard(sonoConfig, audioRecording.WavReader);
+            var decibelArray = SNR.CalculateFreqBandAvIntensity(sonogram.Data, minHz, maxHz, sonogram.NyquistFrequency);
+
+            // prepare plots
+            double intensityNormalisationMax = 3 * decibelThreshold;
+            var eventThreshold = decibelThreshold / intensityNormalisationMax;
+            var normalisedIntensityArray = DataTools.NormaliseInZeroOne(decibelArray, 0, intensityNormalisationMax);
+            var plot = new Plot(speciesName + " Territory", normalisedIntensityArray, eventThreshold);
+            var plots = new List<Plot> { plot };
+
+            //iii: CONVERT decibel SCORES TO ACOUSTIC EVENTS
+            var acousticEvents = AcousticEvent.GetEventsAroundMaxima(
+                decibelArray,
+                segmentStartOffset,
+                minHz,
+                maxHz,
+                decibelThreshold,
+                minTimeSpan,
+                maxTimeSpan,
+                sonogram.FramesPerSecond,
+                sonogram.FBinWidth
+                );
+
+            // ######################################################################
+            acousticEvents.ForEach(ae =>
+            {
+                ae.SpeciesName = speciesName;
+                ae.SegmentDurationSeconds = audioRecording.Duration.TotalSeconds;
+                ae.SegmentStartSeconds = segmentStartOffset.TotalSeconds;
+                ae.Name = abbreviatedSpeciesName;
+            });
+
+            acousticEvents = FilterEventsForSpectralProfile(acousticEvents, sonogram);
+
+            return new RecognizerResults()
+            {
+                Events = acousticEvents,
+                Hits = null,
+                ScoreTrack = null,
+                Plots = plots,
+                Sonogram = sonogram,
+            };
+        }
+
+        /// <summary>
+        /// Remove events whose acoustic profile does not match that of a flying fox.
+        /// </summary>
+        /// <param name="events">unfiltered acoustic events.</param>
+        /// <param name="sonogram">includes matrix of spectrogram values.</param>
+        /// <returns>filtered acoustic events.</returns>
+        private static List<AcousticEvent> FilterEventsForSpectralProfile(List<AcousticEvent> events, BaseSonogram sonogram)
+        {
+            double[,] spectrogramData = sonogram.Data;
+            int colCount = spectrogramData.GetLength(1);
+
+            // The following freq bins are used to demarcate freq bands for spectral tests below.
+            // The hertz values are hard coded but could be included in the config.yml file.
+            int maxBin = (int)Math.Round(8000 / sonogram.FBinWidth);
+            int fourkHzBin = (int)Math.Round(4000 / sonogram.FBinWidth);
```

Comment:
User a better name for `fourkHzBin` and `onekHzBin`. Suggestions:

- `lowerBin`, `middleBin`
- `hertz1000Bin` hertz4000` bin
- `oneKiloHertzBin`, `fourKiloHertzBin`